### ISSUE TITLE
Document usage of -t "" to create fixed field length output in bcp Utility

### ISSUE THREAD

**Issue by Rokartt4209** [MicrosoftDocs/sql-docs#8850]
It would be helpful if the documentation for  "-t " option explicitly showed that field_term can be a double-quoted empty string ("") to instruct BCP to export to fixed-field length format (i.e. no field separators).  A google search reveals people have asked how to use BCP to export to fixed field length files and this simple answer of specifying (space)"" after the -t option does not come up as a solution very often. The examples in the ms documentation show using hex notation( 0x),  escaped character notation (\t), and a single character directly following the -t option (-t,) .   But, never is an example shown, such as "-t """.   

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 86bd520a-7468-fd55-c357-b0f645a917ef
* Version Independent ID: 3dc9dee5-afcb-8287-4441-e84ff717186a
* Content: [bcp Utility - SQL Server](https://learn.microsoft.com/en-us/sql/tools/bcp-utility?view=sql-server-ver16)
* Content Source: [docs/tools/bcp-utility.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/tools/bcp-utility.md)
* Service: **sql**
* Sub-service: **tools-other**
* GitHub Login: @markingmyname
* Microsoft Alias: **maghan**

**Comment by WilliamAntonRohm** [comment]
@Rokartt4209 -- thank you for your detailed feedback.

@markingmyname -- please look into this issue.

**Comment by rwestMSFT** [comment]
@Rokartt4209 We recently updated this article with a refreshed look and feel, and new examples. Do you mind taking a look and letting us know if this issue you reported still needs inclusion?

**Comment by markingmyname** [comment]
@rwestMSFT thank you for addressing this.

@Rokartt4209,

We will now proceed to close this thread. If there are further questions regarding this matter, please tag me in your reply. We will gladly continue the discussion, and we will reopen the issue. To see what the community is wanting, please visit the [SQL Server User Feedback](https://feedback.azure.com/d365community/forum/04fe6ee0-3b25-ec11-b6e6-000d3a4f0da0) site or the [Q & A](https://docs.microsoft.com/en-us/answers/products/sql-server) site.

**Comment by Rokartt4209** [comment]
@markingmyname 

Having read the updated documentation I still would not know how to make BCP write a fixed field length file.  It would be most helpful if there was a specific section on how to do this, such that the file HAS NO FIELD SEPARATORS and each record is terminated with cr/lf. There may be multiple ways of doing it.   I just asked chat.openai.com and I got an answer that says to write the output query to use CAST or CONVERT on each column to CHAR(##) where ## varies for each desired column length and to use the "-c" option on BCP.   This answer is probably wrong, assuming when we say "fixed field length" that we do NOT want any type of field separator character(s).  From the documentation it seems this would still place a tab character between each field.  This answer also differs from what I previously got from chatGPT (which I've noticed is increasingly incorrect on its answers) of using:
 -t ""
What I ultimately did for creating fixed field length output files with cr/lf as record terminator is to define "export" tables in my database where all columns had a fixed CHAR(##) data type. I populated the "export" tables with queries to format all the data into the desired CHAR(##) columns and then used  BCP with an option like:
 -n -C RAW -r\n -T -S
